docs/using-hive.md

```diff
@@ -140,11 +140,9 @@ bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=aws myclu
 Credentials will be read from `~/.azure/osServicePrincipal.json` typically created via the `az login` command.
 
 ```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=azure --azure-base-domain-resource-group-name=myresourcegroup --release-image=registry.svc.ci.openshift.org/origin/release:4.2 mycluster
+bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=azure --azure-base-domain-resource-group-name=myresourcegroup mycluster
```

Comment:
sneaky :)

Reply:
The default release image is a 4.2 image now, so explicitly specifying the release image is no longer necessary.